File: Practice/atcoder/ABC/092/src/c.rb
# ん〜？わからん
# 愚直にやっても間に合う...か...?
n = gets.chomp.to_i
arr = gets.chomp.split.map(&:to_i)
n.times do |i|
  tmp_arr = arr[0, i]+arr[i+1, n-1]
  p tmp_arr
  tmp_v = tmp_arr[0]
  ans = 0
  (0...n-1).to_a.each do |j|
    ans += (tmp_v - tmp_arr[j].abs).abs
  end
  # p ans
  puts ans + (tmp_arr[-1]).abs
end

File: Practice/atcoder/ABC/092/src/c.py
# 差のリストをもっておいて、
# 前後で単調増加/減少する場所にきたら(行き帰りで)2倍して引く

n = int(input())
arr = list(map(int, input().split()))


def dif(arr):
    temp_arr = [0] + arr[:]
    res = []
    for i in range(1, len(temp_arr)):
        res.append(abs(temp_arr[i] - temp_arr[i - 1]))
    return res


lis = dif(arr)
s = sum(lis)+int(arr[-1])
for i in range(n):
    if i != 0 and i != n - 1:
        if arr[]
    else:
